charts/values.yaml

```diff
@@ -0,0 +1,250 @@
+## Default values for charts.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## @param global clusterpedia global config
+global:
+  ## @param dbInstallMode "internal" and "external" are provided
+  ## "internal" means automatic Database installation, default mysql
+  ## "external" means to install the database yourself, must specify the database connection parameters
+  storageInstallMode: "internal"
+  ## persistence configuration
+  ##
+  persistence:
+    ## @param global.persistence.matchNode define the nodeAffinity of pv.
+    ## if defined, will create the pv on the node
+    matchNode: kind-control-plane
+  ## @param global.postgresql.auth define the auth info.
+  postgresql:
+    auth:
+      ## @param auth.postgresPassword Password for the "postgres" admin user
+      ##
+      postgresPassword: "dangerous"
+
+## @param database Select which database backend Clusterpedia will use. Can be 'postgresql' or 'mysql'
+##
+database: postgresql
+## @param installCRDs define flag whether to install CRD resources
+## 
+installCRDs: false
+
+## clusterpedia apiserver config
+apiServer:
+  ## @param apiServer.labels
+  labels: 
+    app: clusterpedia-apiserver
+  ## @param apiServer.replicaCount target replicas
+  replicaCount: 1
+  ## @param apiServer.podAnnotations
+  podAnnotations: { }
+  ## @param apiServer.podLabels
+  podLabels: { }
+  ## @param apiServer.imagePullSecrets
+  imagePullSecrets: []
+  image:
+    ## @param apiServer.image.repository image of the apiserver
+    repository: ghcr.io/clusterpedia-io/clusterpedia/apiserver
```

Comment:
Splitting the `repository` into `registry` and `repository` seems to make it easier to use